packages/core/rpc/coreRpcMethods.ts

```diff
@@ -132,7 +152,7 @@ export class CoreGetFeatures extends RpcMethodType {
       args,
       rpcDriverClassName,
     )
-    const { signal, sessionId, adapterConfig, region, opts } = deserializedArgs
+    const { signal, sessionId, adapterConfig, regions, opts } = deserializedArgs
```

Comment:
converts CoreGetFeatures to use regions (plural) so that we can more easily integrate with renameRegionsIfNecessary, ends up being a natural fit anyways also